papers/yellow/block.tex

```diff
@@ -0,0 +1,67 @@
+\section{Ergo Block Structure}
+Ergo block consists of 4 parts:
+
+\begin{itemize}
+    \item{\em Header } - minimal amount of data, required to synchronize the chain and check PoW correctness.
```

Comment:
"minimal amount of data required to synchronize the chain and check PoW correctness"